### DISCUSSION TITLE
Can you use a single refinement/facet list component to drive multiple result lists?

### DISCUSSION THREAD

**Discussion by tombburnell** [algolia/instantsearch#5011] [Q&A]
I want to have multiple lists of results on my home page “Trending”, “Top Sellers” "editor picks" etc..

And I’d like to have a single multi-select facet control for “Genre” so that when the user selects the genre’s they are interested in, all result lists update to show only products in those selected genres.

I'd also like to store their selections for their next visit.

Is it possible and how would you structure it?

Thanks!

**Comment by Haroenv** [comment]
Indices inherit top-down. So if you have compatible index formats, you can have this layout:

```js
instantsearch().addWidgets([
  refinementList(),
  index().addWidgets([hits()]),
  index().addWidgets([hits()]),
]);
```

**Reply by tombburnell** [reply]
oh cool - how would that look in react?

**Reply by Haroenv** [reply]
In react that looks very similar:

```js
<InstantSearch>
  <RefinementList>
  <Index>
    <Hits/>
  </Index>
  <Index>
    <Hits/>
  </Index>
</InstantSearch>
```

This way the refinements of the refinement list will apply to all indices which are lower in the tree
